app/models/assignment_participant.rb

```diff
@@ -149,52 +149,51 @@ def get_reviewers
 
     reviewers
   end
-  alias_method :reviewers, :get_reviewers
```

Comment:
Maybe we should keep these aliases just in case other code uses the get_x methods.